src/Clock/MockClock.php

```diff
@@ -35,4 +35,14 @@ public function sleep(int $seconds): void
             new DateInterval("PT{$seconds}S")
         );
     }
+
+    public function changeTime(int $seconds): void
```

Comment:
I wonder if we want to refactor this to allow for broader time travel, similar to Laravel?